src/Workspaces/Core/Portable/Formatting/TriviaEngine/AbstractTriviaFormatter.cs

```diff
@@ -818,7 +818,7 @@ private string GetWhitespaceString(LineColumn lineColumn, LineColumnDelta delta)
             var useTabOnlyForIndentation = this.OptionSet.GetOption(FormattingOptions.UseTabOnlyForIndentation, this.Language);
```

Comment:
Yes, I have removed the option. Given that we dont have any test for this option and no more using this option in the code, it is ideal to have this option removed.

Issue #1176 has been opened to track the addition of the option with precise intent and testcases